### PR TITLE
add Asus tinkerboard support

### PR DESCRIPTION
**Pull request by karthikpoduval**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -86,3 +86,7 @@
 	path = sources/meta-elm-binary
 	url = git://github.com/yoedistro/meta-elm-binary
 	branch = master
+[submodule "sources/meta-rockchip"]
+	path = sources/meta-rockchip
+	url = git://git.yoctoproject.org/meta-rockchip
+	branch = dunfell
```
Comment (cbrake) [inline .gitmodules:92]: should this be the master branch in meta-rockchip?

Comment (kraj) [reply .gitmodules:92]: yes it should be `master`

Comment (karthikpoduval) [reply .gitmodules:92]: Thanks @cbrake @kraj, I will fix this.

**Comment by cbrake** [comment]
@karthikpoduval is there anything else you need from us to complete this PR?

**Pull request closed without merging**